doc/sharding.md

```diff
@@ -2,13 +2,13 @@
 
 ### Sharding
 
-Data will be consistently hashed to a set of virtual nodes.  M3DB can be configured to use any hashing function and a configured number of shards.  For the sake of this documentation we’ll choose the following configuration:
+Data is consistently hashed to a set of virtual nodes.  M3DB can be configured to use any hashing function and any number of shards.  For the sake of this documentation we’ll choose the following configuration:
```

Comment:
`Data` is historically the plural of `datum`, but in modern usage it's more commonly treated as a mass noun, i.e. a singular noun regardless of how much of it there is, like bacon. You can have a lot of bacon or a little bacon, but you still say "the bacon is", not "the bacon are".